R/functions.R

```diff
@@ -378,7 +378,7 @@ form_calculation_and_display_limits <- function(data,
   
   #extend display limits to end 
   limits_table <- form_display_limits(limits_table = limits_table, 
-                                      counter = counter_at_period_start + periodMin + 1,
```

Comment:
This is a slightly different variant of the same problem, in that here we want the counter for the start of the display limits to be the point immediately after the end of the calculation period. So with periodMin = 21 and counter_at_period_start = 22, the correct start of the display limits should be 43, but this was giving 44. That would have been "right" for the fact that with the old counter system (i.e. current version on master) the calculation period here would actually be of length `periodMin + 1`, but with the correct to that, this also needs correcting.